qemu/tests/cfg/rebase_onto_no_backing_file.cfg

```diff
@@ -0,0 +1,32 @@
+- rebase_onto_no_backing_file:
+    only raw
+    virt_test_type = qemu
+    type = rebase_onto_no_backing_file
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    # md5sum binary path
+    md5sum_bin = "md5sum"
+    image_chain = "image1 sn"
+    image_format_image1 = "raw"
+    image_name_sn = "images/sn"
+    # set size to "", so during snapshot creation
+    # the cmdline will not have specified size option
+    image_size_sn = ""
+    image_format_sn = qcow2
+    tmp_dir = /var/tmp
+    tmp_file_name = ${tmp_dir}/testfile
+    file_create_cmd = "dd if=/dev/urandom of=%s bs=1M count=512"
+    guest_file_name = ${tmp_file_name}
+    Windows:
+        guest_file_name = C:\testfile
+        x86_64:
+            sync_bin = WIN_UTILS:\Sync\sync64.exe /accepteula
+        i386, i686:
+            sync_bin = WIN_UTILS:\Sync\sync.exe /accepteula
+    variants:
+        - compat_level:
+            variants:
+                - compat_0.10:
+                    image_extra_params = "compat=0.10"
```

Comment:
Based on my knowledge, the scenario could be tested on RHEL-6 without specifying the version of qcow2 image format. But it is better to check whether this case need to be supported on rhel-6.

Reply:
@hereischen @hellenmao qcow2v3 was supported since qemu-1.1.
Detail information could be checked from upstream changelog as below
https://wiki.qemu.org/ChangeLog/1.1

Reply:
required_qemu = [1.1, )